### ISSUE TITLE
Curator does not support Elasticsearch 6.0.0-beta1

### ISSUE THREAD

**Issue by algestam** [elastic/curator#1030]
When running the latest version of curator (5.1.2) against an Elasticsearch instance running 6.0.0-beta1, the following error is produced:

```
2017-08-10 12:14:55,923 ERROR     Elasticsearch version 6.0.0 incompatible with this version of Curator (6.0.0)
Traceback (most recent call last):
  File "/usr/bin/curator", line 9, in <module>
    load_entry_point('elasticsearch-curator==5.1.2', 'console_scripts', 'curator')()
  File "/usr/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/curator/cli.py", line 167, in cli
    client = get_client(**client_args)
  File "/usr/lib/python2.7/site-packages/curator/utils.py", line 788, in get_client
    'Error: {0}'.format(e)
elasticsearch.exceptions.ElasticsearchException: Unable to create client connection to Elasticsearch.  Error: Elasticsearch version 6.0.0 incompatible with this version of Curator (6.0.0)
```

**Comment by untergeek** [comment]
This is a known issue.  It will be a bit before I'm able to get a 6.0 beta out for Curator.

**Comment by algestam** [comment]
Ok, thanks for the info 👍

**Comment by shaharmor** [comment]
Any update on this?

**Comment by untergeek** [comment]
No.  I'm kind of dependent on at least a pre-release of the 6.0 python client before I can alter the current code to work with ES 6.

**Comment by mayoung8** [comment]
@untergeek, would you expect curator 6.0 to be available with elasticsearch 6.0 GA release?

**Comment by untergeek** [comment]
Actually, I've made Curator 5.4 completely compatible with 6.0.0-rc2, and it will be out shortly.

**Comment by untergeek** [comment]
This is resolved now, with the release of Curator 5.4.0
